install/install-community-operators.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
This one will install the community versions of Strimzi and Prometheus if the user hasn't already installed them. I'm a little unsure how to deal with this one because a tech preview user likely won't be using the community Strimzi in practice.

Reply:
More assumptions on my part: assuming we'd be deploying (and referring in the docs) to streams operators downstream. The way I've written the connection procedure is that you use this script to deploy AMQ Streams or configure your current deployment of AMQ Streams. The doc can bend to any realities for the tech preview...

Reply:
What do you think about calling it `install-dependency-operators.sh` ? Of all the scripts, only `console-install.sh` is required, with the others being optional depending on the current state of the user's cluster.